### ISSUE TITLE
Discord crashing when I open dms or any kind of server[Bug]

### ISSUE THREAD

**Issue by ExiTheLich** [rauenzi/BetterDiscordApp#557]
**Describe the Bug**
Whenever I open a dm or server, the app instantly crashes and I can't restart it.

**To Reproduce**
1) Open discord
2) Click on a dm or server
3) Crash

**Expected Behavior**
I expect it to open the dm or server without crashing, as i have been using BD for a while and haven't had this happen until now

**Screenshots**
![image](https://user-images.githubusercontent.com/78621701/107089736-6b553600-67ff-11eb-958b-07748252ae4b.png)

**Discord Version**
Stable

**Additional Context**
It's not just for a specific person or server, any server or any person leads to the same result

**Comment by chiumax** [comment]
Same here

**Comment by chiumax** [comment]
```js
TypeError: Cannot read property 'getChannel' of null
    at Object.getRoleColor (remote.js:1)
    at MessageContent (remote.js:1)
    at _a (78c53dd4b4663af488c4.js:2)
    at Ls (78c53dd4b4663af488c4.js:2)
    at yu (78c53dd4b4663af488c4.js:2)
    at Iu (78c53dd4b4663af488c4.js:2)
    at cu (78c53dd4b4663af488c4.js:2)
    at 78c53dd4b4663af488c4.js:2
    at t.unstable_runWithPriority (78c53dd4b4663af488c4.js:2)
    at zi (78c53dd4b4663af488c4.js:2)
    at Ji (78c53dd4b4663af488c4.js:2)
    at Qi (78c53dd4b4663af488c4.js:2)
    at ce (78c53dd4b4663af488c4.js:2)
    at Dn (78c53dd4b4663af488c4.js:2)
    at Mn (78c53dd4b4663af488c4.js:2)
    at Pn (78c53dd4b4663af488c4.js:2)
    at t.unstable_runWithPriority (78c53dd4b4663af488c4.js:2)
    at Cn (78c53dd4b4663af488c4.js:2)
    at HTMLDocument.r (3609a027dbe02c276699.js:2)
(anonymous) @ 3609a027dbe02c276699.js:2
Es @ 78c53dd4b4663af488c4.js:2
Rs.o.componentDidCatch.n.callback @ 78c53dd4b4663af488c4.js:2
yo @ 78c53dd4b4663af488c4.js:2
So @ 78c53dd4b4663af488c4.js:2
bu @ 78c53dd4b4663af488c4.js:2
t.unstable_runWithPriority @ 78c53dd4b4663af488c4.js:2
zi @ 78c53dd4b4663af488c4.js:2
Nu @ 78c53dd4b4663af488c4.js:2
cu @ 78c53dd4b4663af488c4.js:2
(anonymous) @ 78c53dd4b4663af488c4.js:2
t.unstable_runWithPriority @ 78c53dd4b4663af488c4.js:2
zi @ 78c53dd4b4663af488c4.js:2
Ji @ 78c53dd4b4663af488c4.js:2
Qi @ 78c53dd4b4663af488c4.js:2
ce @ 78c53dd4b4663af488c4.js:2
Dn @ 78c53dd4b4663af488c4.js:2
Mn @ 78c53dd4b4663af488c4.js:2
Pn @ 78c53dd4b4663af488c4.js:2
t.unstable_runWithPriority @ 78c53dd4b4663af488c4.js:2
Cn @ 78c53dd4b4663af488c4.js:2
r @ 3609a027dbe02c276699.js:2
```

**Comment by Strencher** [comment]
Paste this here in your console (**Ctrl + shift + i** => "Console" tab) `BdApi.disableSetting("bda-gs-7")`to disable ColoredText and stop the crashes. Make sure you reload your client with `Ctrl + R`.

**Comment by chiumax** [comment]
@Strencher god bless

**Comment by MacGyverPY** [comment]
same thing happens to me... ill disable that setting and see... https://github.com/rauenzi/BetterDiscordApp/issues/556

**Comment by rauenzi** [comment]
Fixed by #573
